transforms/universal/ededup/ray/src/ededup_transform_ray.py

```diff
@@ -81,7 +81,7 @@ def _process_cached_hashes(self, hd: dict[str, str]) -> list[str]:
         return unique
 
 
-class EdedupRuntime(DefaultRayTransformRuntime):
```

Comment:
Of course for backwards compatibility you should leave this class somehow

Reply:
It is internal, so there is no compatibility issues